src/BlockTypes/AddToCartForm.php

```diff
@@ -96,7 +96,7 @@ protected function render( $attributes, $content, $block ) {
 
 		$classname          = $attributes['className'] ?? '';
 		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
-		$product_classname = $is_descendent_of_single_product_block ? 'product' : '';
+		$product_classname  = $is_descendent_of_single_product_block ? 'product' : '';
```

Comment:
I know this is unrelated and you have opened a separate PR to add this. I think it's fine to leave it here. 👍